### PR TITLE
flathub: add com.tdameritrade.ThinkOrSwim

### PR DESCRIPTION
**Pull request by jkoelker**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by hfiguiere** [comment]
bot, build com.tdameritrade.ThinkOrSwim

**Comment by hfiguiere** [comment]
```
• url-not-found         : <screenshot> failed to download (HTTP 404: Not Found) [https://www.tdameritrade.com/retail-en_us/resources/images/tools-and-platforms/thinkorswim/rebrand/tos_tabbedcomponent_controlmarket_810x469_light.png]
```

Screenshot isn't found.

**Comment by jkoelker** [comment]
Yea sorry, I'll find where they host the new ones and update in a moment ;(.

**Comment by jkoelker** [comment]
I switched to one hosted on the opensuse wiki, as that's probably a more stable URL.

**Comment by hfiguiere** [comment]
bot, build com.tdameritrade.ThinkOrSwim

File: com.tdameritrade.ThinkOrSwim.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+tags:
+  - proprietary
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+
+finish-args:
+  - --persist=.thinkorswim
+  - --socket=x11
```
Comment (nedrichards) [inline com.tdameritrade.ThinkOrSwim.yaml:17]: can you add --share=ipc as well with socket=x11 to access x11 acceleration. (see https://docs.flatpak.org/en/latest/sandbox-permissions.html for more)

Comment (jkoelker) [reply com.tdameritrade.ThinkOrSwim.yaml:17]: Added, thanks for the pointer \o/

**Comment by hfiguiere** [comment]
bot, build com.tdameritrade.ThinkOrSwim

**Pull request closed without merging**